<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirScanWidget</class>
 <widget class="QWidget" name="DirScanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,96">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Directory scan</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_dir_edit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_dir_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
     <item>
      <widget class="QToolButton" name="m_start_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../qrc/meduzzza.qrc">
         <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_stop_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qrc/meduzzza.qrc">
         <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="m_scan_view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc/meduzzza.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_dir_button</sender>
   <signal>clicked()</signal>
   <receiver>DirScanWidget</receiver>
   <slot>dirClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_start_button</sender>
   <signal>clicked()</signal>
   <receiver>DirScanWidget</receiver>
   <slot>startClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_stop_button</sender>
   <signal>clicked()</signal>
   <receiver>DirScanWidget</receiver>
   <slot>stopClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dirClickedSlot()</slot>
  <slot>startClickedSlot()</slot>
  <slot>stopClickedSlot()</slot>
 </slots>
</ui>
